green-shop/source/new.pug

```diff
@@ -0,0 +1,58 @@
+extends layout/_default
+
+block head
+  +head('Green Shop | Giỏ hàng')
+
+block body
+  block header
+    include layout/_header
+  block main
+    .main.product-new
+      .product-new__wrapper.row
+        +breadcrumb(
+        ['Home', 'Cây dạ lam'],
+        ['home', 'new'])
+        .product-new__sidebar-left.col-12.col-md-3
```

Comment:
Phần này theo ý a sẽ làm như này nhé, 
.sidebar.sidebar-left.product-new__sidebar
sidebar để sử dụng cho phần chung toàn site
sidebar-left chỉ có ở bên left và tác dụng toàn site
product-new__sidebar Chỉ có tác dụng ở bên sidebar của mỗi trang product new 
Sau này ở các trang khác mà sử dụng lại các dc nhiều nhất